app/views/mortgage_calculator/stamp_duties/_stamp_duty_tip.html.erb

```diff
@@ -1,7 +1,22 @@
 <div class="mortgagecalc__panel--full">
   <div ng-show="expandedStampDutyInformation" class="mortgagecalc__tip">
     <div class="stamp-duty stamp-duty__calculation-explanation">
-      <%= render 'calculation_explanation' %>
+      <% if first_time_buyer? %>
+        <div ng-hide="js" class="stamp-duty__explanation-firsttimebuyer ng-hide is-active">
+          <%= render 'first_time_buyer_explanation' %>
+        </div>
+      <% else %>
+        <div ng-hide="js" class="stamp-duty__explanation-nexthome ng-hide is-active">
+          <%= render 'next_home_buyer_explanation' %>
+        </div>
+      <% end %>
+
+      <div class="rendered-from-js stamp-duty__explanation-firsttimebuyer">
+        <%= render 'first_time_buyer_explanation' %>
+      </div>
+      <div class="rendered-from-js stamp-duty__explanation-nexthome">
+        <%= render 'next_home_buyer_explanation' %>
+      </div>
     </div>
   </div>
-</div>
+</div>
```

Comment:
new line or is that a github anomaly?

Reply:
👍  for use of partials on the stamp duty tip view

Reply:
@margo it's not a new. There's no new line at the end of the file.